mongodb_migrations/cli.py

```diff
@@ -30,7 +30,7 @@ def run(self):
             self.config.mongo_database)
         files = os.listdir(self.config.mongo_migrations_path)
         for file in files:
-            result = re.match('^(\d+)[_a-z]*\.py$', file)
+            result = re.match('^[_a-zA-Z0-9]+_[_a-zA-Z0-9]*\.py$', file)
```

Comment:
1. Hash, if it is a hex value, can only have characters ranging from (0-9 + a-f)
2. Either the convention is to use uppercase, or lowercase. Hash with mixed case values looks quite odd